pkg/ee/metering/report_config_handler.go

```diff
@@ -92,6 +97,18 @@ func (m createReportConfigurationReq) Validate() error {
 		}
 	}
 
+	if m.Body.Types != nil {
+		if len(*m.Body.Types) == 0 {
+			return utilerrors.NewBadRequest("at least one report type is required")
```

Comment:
should we leave some default type? Like the report type that was before set before we added these options?

Reply:
Default types are applied when no `types` are provided (`...Types == nil`) in the incoming request. In case someone implicitly set `types: []`, then we want to throw an error. Or you mean sth else?

Reply:
I meant if we default for nil, why not default for empty list as well?

Reply:
But I dont have strong  feelinga about that, just wondering

Reply:
I would prefer to leave an error when someone explicitly tries to set na empty list `[]`. Otherwise someone may think that no reports will be generated but instead we will generate both cluster and namespace reports.